src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -72,6 +72,23 @@ public class BiomeGUI extends ModElementGUI<Biome> {
 	private final JSpinner heightVariation = new JSpinner(new SpinnerNumberModel(0.2, 0, 2, 0.1));
 	private final JSpinner temperature = new JSpinner(new SpinnerNumberModel(0.5, -1.0, 2.0, 0.1));
 
+	//m STANDS FOR MULTI-NOISE
+	private final JCheckBox usePureMultiNoise = L10N.checkbox("elementgui.common.enable");
+	private final JSpinner mTemperature1 = new JSpinner(new SpinnerNumberModel(0.0, -1.0, 1.0, 0.05));

```

Comment:
Same thing for them (use the same names)

Reply:
This was fixed in "First Fix" (well, at least I hope so)